src/manage_sql.c

```diff
@@ -34209,7 +34211,7 @@ create_credential (const char* name, const char* comment, const char* login,
     }
 
   /* SNMP passwords */
-  if (community)
+  if (community || using_snmp_v3)
```

Comment:
The `lsc_crypt_encrypt` function call setting the `community` within the `if (community || using_snmp_v3)` block should handle the community parameter either as as a separate case that only sets password and privacy_password or by using an empty string for `community` is if it is `NULL`.

Reply:
Sorry, I've mixed this up with the entry **name** being NULL as a terminator, so this should be okay.